index.md

```diff
@@ -37,28 +41,70 @@ img: ":home-heading.jpg"
+   </div>
+   <div class="containerMember">
+       <img class="imgMembers" id="imgMember2" src="/assets/img/home/kittyRes.jpg" />
+       <h4>Kitty Brandon-James</h4>
+       <h2 class="job">Educational Consultant & Curator</h2>
+       <p id="desc">Kitty is an experienced researcher, university lecturer, and curator with a special interest in multisensory pedagogies, play, and thinking through doing. She runs action research about teaching for creativity, and firmly believes that thinking through objects, exhibitions, and curating contains great potential to escape digital silos, enhance empathy, problem solve, and connect multiple social, political, and cultural perspectives. Trained as an art historian and carpenter, her specialism is art produced in Moscow in 2012–21. She teaches at University College London.
+       Kitty has a BA in History from King’s College London, an MA in Contemporary Art & Politics from the Courtauld Institute of Art, University of London, a Post-Graduate Certificate in Education from the University of Cambridge, and is currently working towards her PhD in Russian Art and Politics at the School of Slavonic Studies, UCL.
+</p>
+   </div>
+   <div class="containerMember">
+       <img src="/assets/img/home/icon.png" />
+       <h4>Jule Rubi</h4>
+       <h2 class="job">Cultural Projects Manager and Curator</h2>
+        <p id="desc">Description</p>
+   </div>
+   <div class="containerMember">
+       <img class="imgMembers" id="imgMember3" src="/assets/img/home/caroRes.jpg" />
+       <h4>Caroline Trotman</h4>
+       <h2 class="job">Historian and Curator</h2>
+        <p id="desc">Caroline is a motivating and analytical teacher of History, Geography, Geopolitics and Political Science in Paris, France. In 2009-15, she was based between Florida and California, USA, then London and Oxford, UK, where she worked as an Assistant Curator and Registrar at various museums, which is how she met Elisa. Together, they went on to collaborate on a research project on the Buzludzha monument in Bulgaria in 2014-16, from which a book, articles and an exhibition emerged.
+        Caroline has a BA in History, Art History & Geography from the Sorbonne, Paris, and three MAs: in Modern Art & English from Oxford Brookes University, in Museum Studies from the University of Leicester, and Contemporary Art Theory from Goldsmiths, University of London (all UK).
+</p>
+   </div>
+   <div class="containerMember">
+       <img src="/assets/img/home/icon.png" />
+       <h4>Laura Garrido</h4>
+       <h2 class="job">AV & New Technologies Specialist</h2>
+        <p id="desc">Description</p>
+   </div>
+   <div class="containerMember">
+       <img class="imgMembers" id="imgMember4" src="/assets/img/home/susana.jpg" />
+       <h4>Susana Miralles</h4>
+       <h2 class="job">Business Manager</h2>
+        <p id="desc">Description</p>
+   </div>
+</div>
```

Comment:
I assume the PO has reviewed this new text? Like have you shared a screengrab of how it looks with them?